shakenfist/daemons/net.py

```diff
@@ -94,12 +95,57 @@ def _maintain_networks(self):
         db.persist_node_vxid_mapping(
             config.parsed.get('NODE_NAME'), vxid_to_mac)
 
+    def _process_network_node_workitems(self):
+        workitem = db.dequeue('networknode')
+        if not workitem:
+            time.sleep(0.2)
```

Comment:
Could the SF etcd library use watch_prefix() to avoid this busy wait?
I acknowledge this would mean a separate thread for _maintain_networks() but better than a etcd request every 0.2 secs?

Reply:
I think this is a good idea. I am hesitant to do it now though because the code feels quite instable to me... Later in this series (which you haven't seen yet) I discover that we are using different names for locks and are clashing, and move instance create / delete to queues too. I think I'd like to get to somewhere a little more stable before optimising this code -- especially because we might abandon this approach entirely when we discover we can't make it stable or something. Is that ok to defer for now with a bug or something?

Reply:
sounds fair to me: "bug or something"